src/main/java/de/slimecloud/slimeball/features/staff/absence/AbsenceTable.java

```diff
@@ -0,0 +1,30 @@
+package de.slimecloud.slimeball.features.staff.absence;
+
+import de.mineking.databaseutils.Table;
+import de.mineking.databaseutils.Where;
+import net.dv8tion.jda.api.entities.UserSnowflake;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+
+public interface AbsenceTable extends Table<Absence> {
+
+	default Absence addAbsence(@NotNull Absence absence) {
+		insert(absence);
```

Comment:
You should probably not use `Absecense` as parameter here but instead `@NotNull Member` and `@Nullable Instant` and then call the constructor in here (See every other table)